app/src/org/commcare/views/widgets/CommCareAudioWidget.java

```diff
@@ -138,12 +138,6 @@ public void onRecordingCompletion(String audioFile) {
         recordingNameText.setText(Localization.get("recording.custom"));
     }
 
-    @Override
-    public String getFileUniqueIdentifier() {
-        String formFileName = FormEntryInstanceState.mFormRecordPath.substring(FormEntryInstanceState.mFormRecordPath.lastIndexOf("/") + 1);
-        return formFileName + "_" + mPrompt.getIndex().toString() + "_" + (new Date().getTime());
```

Comment:
The fileName returned by this is really weird. It has a `.xml`and `,`(a comma if mPrompt.getIndex returns something like 2,3).